test/types/customType-contentRelationship.types.ts

```diff
@@ -96,6 +96,17 @@ expectType<prismic.CustomTypeModelContentRelationshipField<string, "foo">>({
 	},
 })
 
+/**
+ * Supports optional `text` property.
+ */
+expectType<prismic.CustomTypeModelContentRelationshipField>({
+	type: prismic.CustomTypeModelFieldType.Link,
+	config: {
+		label: "string",
+		select: prismic.CustomTypeModelLinkSelectType.Document,
+	},
+})
+
```

Comment:
⚠️ #issue: This test can be removed since we no longer check for the `text` property. This test is identical to the file's first `expectType` test.